packages/components/src/tabs/styles.ts

```diff
@@ -151,9 +151,11 @@ export const StyledTabList = styled( Ariakit.TabList )`
 export const Tab = styled( Ariakit.Tab )`
 	& {
 		/* Resets */
+		appearance: auto;
 		border-radius: 0;
 		background: transparent;
 		border: none;
+		box-sizing: border-box;
```

Comment:
This is effectively needed to make sure that the tab indicator measures the tab size correctly